Stepippo/Classes/Controllers/ProgVC.swift

```diff
@@ -23,21 +23,42 @@ class ProgVC: UIViewController {
         print(sender.isChecked)
     }
     
+    @IBAction func showActionSheet() {
+        // アクションシートのタイトルとメッセージ
+        let actionSheet = UIAlertController(title: "期間変更", message: "達成したい目標期間を選択してください", preferredStyle: .actionSheet)
+        
+        // 選択肢と、選択肢ごとの処理
+        // handlerで指定した処理が、ボタン押下時に行われる
+        let period1 = UIAlertAction(title: "毎日", style: .default, handler: actionSheetChoose(sender:))
+        let period2 = UIAlertAction(title: "毎週", style: .default, handler: actionSheetChoose(sender:))
+        let period3 = UIAlertAction(title: "毎月", style: .default, handler: actionSheetChoose(sender:))
+        let cancel = UIAlertAction(title: "キャンセル", style: .cancel, handler: {
+            (action: UIAlertAction) -> Void in
+            print("キャンセルしました")
+        })
+        // アクションシートに、定義した選択肢を追加する
+        actionSheet.addAction(period1)
+        actionSheet.addAction(period2)
+        actionSheet.addAction(period3)
+        actionSheet.addAction(cancel)
+        
+        // アクションシートを表示する
+        present(actionSheet, animated: true, completion: nil)
+    }
+    
+    // アクションシートで選択した時の処理
+    func actionSheetChoose(sender: UIAlertAction) {
```

Comment:
他のメソッドもですが、他のクラスから使わないメソッドやプロパティにはprivateをつけておきましょう🙆‍♂️
privateをつける利点については検索するとたくさん出てくるかと思います👌
```suggestion
    private func actionSheetChoose(sender: UIAlertAction) {
```

Reply:
praivateの意味は理解できました💡
なんでそうする必要があるのかがまだわかっていないので
引き続き調べてみようと思いますっ

Reply:
具体的には、他のクラスから使わない（ProgVCクラス内でしか使わない）変数や関数はprivateにましょう👌

Reply:
ありがとうございます^0^！
少しずつ判断できるスピードを上げていきたいと思います^-^♪